appservice/src/connectToGitHub.ts

```diff
@@ -63,9 +63,12 @@ export async function connectToGitHub(node: AzureTreeItem, client: SiteClient, c
         isMercurial: false
     };
 
+    // tslint:disable-next-line:no-non-null-assertion
+    const repoName: string = siteSourceControl.repoUrl!.substring('https://github.com/'.length);
```

Comment:
How do you get the org name when showing the org quick pick? There has to be a simpler (and more reliable) way to do this...